packages/web-twig/src/Resources/components/UNSTABLE_EmptyState/UNSTABLE_EmptyStateSection.twig

```diff
@@ -0,0 +1,17 @@
+{# API #}
+{%- set props = props | default([]) -%}
+{%- set _children = block('content') -%}
+
+{# Class names #}
+{%- set _rootClassName = _spiritClassPrefix ~ 'UNSTABLE_EmptyState__section' -%}
+
+{# Miscellaneous #}
+{%- set _styleProps = useStyleProps(props) -%}
+{%- set _classNames = [ _rootClassName, _styleProps.className ] -%}
+
+{% embed "@spirit/stack.twig" with { props: props | merge({
+    UNSAFE_className: _classNames | join(' '),
+    UNSAFE_style: _styleProps.style,
+}) } %}
+    {% block content %}{{ _children | raw }}{% endblock %}
```

Comment:
Here is the potential for **Cross Site Scripting** (_XSS_).

Reply:
refactored